reindexer/complete_reindex/src/requirements.txt

```diff
@@ -1 +0,0 @@
-wellcome_aws_utils==2.0.1
```

Comment:
You need to compile the `requirements.txt` (I think there's a Make task for it), or the deps won't end up in the image.

Reply:
Might have to show me where this is, sorry.

Reply:
```console
$ cd ~/repos/platform
$ make $(pwd)/reindexer/complete_reindex/src/requirements.txt
```